spec/models/work_version_spec.rb

```diff
@@ -85,6 +85,25 @@
     end
   end
 
+  describe '#globus_endpoint_fullpath' do
+    subject(:work_version) { create(:work_version, work: create(:work)) }
+
+    before do
+      work_version.globus_endpoint = globus_endpoint
```

Comment:
This can't be patched into line 89 or we get a stack too deep error, as the globus endpoint needs to know the work's ID and the work version's version but that's not known until *after* line 89 fires.